docs/samples/v1beta1/torchserve/proto/management.proto

```diff
@@ -0,0 +1,117 @@
+syntax = "proto3";
```

Comment:
TorchServe already have the gRPC proto files defined [here](https://github.com/pytorch/serve/tree/master/frontend/server/src/main/resources/proto). Why do we need to duplicate this in the KFS side?